drones/common/movement_instruction.py

```diff
@@ -0,0 +1,26 @@
+"""Movement instruction bundle
+"""
+
+from dataclasses import dataclass
+from typing import List
+
+import drone_instruction as di
+
+
+@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
```

Comment:
There is no need for overriding default parameters in dataclass e.g. frozen is set to False by default.